### DISCUSSION TITLE
Configuration of postgres db with conductor

### DISCUSSION THREAD

**Discussion by srp-asach** [Netflix/conductor#3421] [Q&A]
Looking for assistance configuring postgres db with conductor

1. In which file configurations needs to be added

**Comment by v1r3n** [comment]
@srp-asach we publish a build that uses postgres + redis, if you are interested, checkout https://github.com/orkes-io/orkes-conductor-community

**Reply by srp-asach** [reply]
I want to go without docker. Postgres is installed locally. I want to point it to this instance.

**Reply by manan164** [reply]
Hi @srp-asach ,
Here is the properties for postgres configuration,
```
conductor.db.type=postgres
spring.datasource.url=jdbc:tc:postgresql:11.15-alpine:///conductor
spring.datasource.username=postgres
spring.datasource.password=postgres
spring.datasource.hikari.maximum-pool-size=8
spring.datasource.hikari.auto-commit=false
spring.flyway.locations=classpath:db/migration_postgres
```
Taken from, https://github.com/Netflix/conductor-community/blob/main/persistence/postgres-persistence/src/test/resources/application.properties
